index.html

```diff
@@ -10,8 +10,13 @@
     <link rel="stylesheet" type="text/css" href="styles/main.css">
 </head>
 <body __wid="TDi4iuCp">
-<div id="d3_container" __wid="iZUBjGJV">
-    <svg class="canvas" id="canvas" __wid="PDgACC8e"></svg>
-</div>
+    <div id="CogCanvas">
+        <div id="d3_container" __wid="iZUBjGJV">
+            <svg class="canvas" id="canvas" __wid="PDgACC8e"></svg>
+        </div>
+        <div id="content_container">
```

Comment:
It's better to use JS to generate this div when it's initialized. I'll change the canvas correspondingly.

Reply:
Left Undone For future work.